### ISSUE TITLE
For new alert channel page, webhook URL field should be made mandatory is slack is chosen as the alert channel

### ISSUE THREAD

**Issue by pranay01** [SigNoz/signoz#391]
For slack to work as a notification channel, a webhook URL is necessary. Users should not be able to save the alert channel without providing webhook URL.

Users should be prompted to enter webhook URL if they try to save a new alert channel without providing webhook URL
![Image 22-11-2021 at 12 27](https://user-images.githubusercontent.com/504541/142817412-c43cc792-13b9-44d2-a144-070e699329d8.jpg)

**Comment by shackattackk** [comment]
Hi @pranay01, Can I try working on this one?

**Comment by pranay01** [comment]
sure @shackattackk. Assigned to you

Also - users should see a message above the Webhook URL field if they try saving Alert channel without webhook URL 

![Screenshot 2021-11-22 at 9 40 47 PM](https://user-images.githubusercontent.com/504541/142895833-5941004a-3c57-478a-9d1c-b67dbd23e689.png)

**Comment by shackattackk** [comment]
will this be good enough?
![image](https://user-images.githubusercontent.com/10613992/142960858-cb0bc739-4a75-48cf-8bb2-3520c450c267.png)

**Comment by pranay01** [comment]
Yeah, seems good. Can you also share a video on how this works - i.e. what happens if user clicks Save button without providing webhook URL? A quick loom video/ any other video would be good

@pal-sig Please check this PR

**Comment by shackattackk** [comment]
cant seem to make it work properly. Will remove myself as assignee so that others can work on this as well.

**Comment by shashank99928** [comment]
I can work on this issue

**Comment by shashank99928** [comment]
Is it alright if  I disable the button when webhook url is not entered?

**Comment by pranay01** [comment]
yeah @shashank99928 - disabling the button when webhook URL is not entered should be fine. Were you able to make any progress on this issue?
